features/features.go

```diff
@@ -136,6 +136,8 @@ const (
 	CoinVersion1_2Feature = "coin:v1.2"
 	// Enables minting & burning via Binance Gateway
 	CoinVersion1_3Feature = "coin:v1.3"
+	// Enables minting & burning via Binance Smartchain Gateway
+	CoinVersion1_4Feature = "coin:v1.4"
```

Comment:
Considering the number of references to this feature in the gateway contract `TGVersion1_7 = "tg:v1.7"` probably would've been a better name.